src/dbnode/storage/bootstrap/process.go

```diff
@@ -215,23 +219,24 @@ func (b bootstrapProcess) Run(
 			},
 		})
 		secondRanges := b.newShardTimeRanges(
-			dataRanges.secondRangeWithPersistFalse.Range, namespace.Shards)
+			dataRanges.secondRange.Range, namespace.Shards)
```

Comment:
Hm why are we using “secondRange” instead of “secondRangeWithPersistFalse” here?